README.md

```diff
@@ -1714,12 +1714,13 @@ end
 
 Parameters with types that support `#length` method can be restricted to have a specific length with the `:length` option.
 
-The validator accepts `:min` or `:max` or both options to validate that the value of the parameter is within the given limits.
+The validator accepts `:min` or `:max` or both options to validate that the value of the parameter is within the given limits,
+In addition, if need to allow `nil` value, please to pass the `allow_nil: true` option.
```

Comment:
For another PR, I think we're being too polite by saying `please` in the documentation. It's just a reference, no need to be nice :)